proto/aclk/v1/envelope.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package aclk.v1;
+
+import "proto/chart/v1/instance.proto";
+import "proto/chart/v1/dimension.proto";
+
+option go_package = "aclk/v1;aclkenvelope";
+
+// ACLKMessageEnvelope is the wrapper used by the Agent to dispatch bundled, sequenced messages to the Cloud
+message ACLKMessageEnvelope {
```

Comment:
I worry about misleading name here as this is Chart/Dim update only.... It is not an envelope used for every message. What do you think guys?

Reply:
Indeed you are correct @underhood  it is not an envelope for all messages. Since it is the envelope for chart messages, would it make sense to add also the `Chart` suffix after the `ACLK` part?

So for instance, the message could be called `ACLKChartMessageEnvelope`

If it does not make much sense, we could consider dropping the envelope altogether since it is used only by 2 new Cloud messages (Chart/Dim)

Reply:
If we drop it which can make sense as original purpose was to have common msg envelope.... do we have 2 different topics to send the msgs? @stelfrag do we need 1 message to send both charts and dims?

Reply:
We discussed about using one _potentially_ "fat" message (sent to one topic) including a collection of `ChartInstance` updates and a collection of `ChartDimension` updates. Is this correct?

Reply:
I think that one possible solution could be to:

- rename this message to `ChartsAndDimensionsUpdated`
- move it in the charts domain

Reply:
>Ok lets not complicate I guess and just rename the message for clarity. I will continue my fight with protoc which somehow manages to generate crazy includes in the generated code for those combined messages.

>I think that one possible solution could be to:
rename this message to ChartsAndDimensionsUpdated
move it in the charts domain

Ok So I think we agree on that. I'll make the appropriate changes and then merge this. 
I'll let you pick up your fight where you left it :)
Be strong, `protoc` also messed up with my nerves the previous day